src/modules/endpoint/reducers/pushEvent.js

```diff
@@ -13,9 +13,7 @@ const pushEvent = (state: CityContentStateType, action: PushEventActionType): Ci
 
   const getEventRoute = (): EventRouteStateType => {
     if (!path) {
-      const allAvailableLanguages = new Map(cityLanguages.map(
-        language => [language.code, `/${city}/${language.code}/events`]
-      ))
+      const allAvailableLanguages = new Map(cityLanguages.map(lng => [lng.code, null]))
```

Comment:
I think the null is a bit odd here. This means that the list of events should be displayed right?

Reply:
Not really intuitive imo, but should be fine. For me this looks more like the language is not available, even though it is called `allAvailableLanguages` so it should be fine.

Reply:
Yeah, but it's consistent with the `eventRouteConfig`. We can consider changing it there as well, though, but I'd extract that to another task.